### ISSUE TITLE
More books needed

### ISSUE THREAD

**Issue by TeMPOraL** [LispLang/lisplang.github.io#23]
Some missing:
- PAIP
- AMOP
- Common Lisp Recipes (there's a pull request for that already)

**Comment by eudoxia0** [comment]
Yes, this is definitely an important thing to cover.

**Comment by chuchana** [comment]
[Common Lisp - A Gentle Introduction to Symbolic Computation (David S. Touretzky)](http://www.cs.cmu.edu/~dst/LispBook/) is also missing.

Adding information on the books style in general and on whether it contains exercises (and solutions) would also make sense in my opinion.

Some more that are available freely on the web:
- [Common Lisp the Language, 2nd Edition (Guy L. Steele)](https://www.cs.cmu.edu/Groups/AI/html/cltl/cltl2.html)
- [Successful Lisp - How to Understand and Use Common Lisp (David B. Lamkins)](https://www.psg.com/~dlamkins/sl/cover.html)
- [COMMON LISP - An Interactive Approach (Stuart C. Shapiro)](https://www.cse.buffalo.edu/~shapiro/Commonlisp/)

**Comment by eudoxia0** [comment]
>Adding information on the books style in general and on whether it contains exercises (and solutions)

Yes. Maybe a few cute colorful tags like 'Intro', 'Advanced', 'Domain-Specific'.

**Comment by chuchana** [comment]
Sounds good.

One more missing book is Object-Oriented Programming in Common Lisp (Sonya E. Keene).

**Comment by eudoxia0** [comment]
Yes, yes, there's plenty of books left :grimacing:

**Comment by chuchana** [comment]
Does it make sense to add some of the older ones as a list? I cannot find high quality cover images for some of them.

**Comment by eudoxia0** [comment]
You mean, as a separate list? I don't think so. If high-res covers can't be found, we can just use smaller ones. A few blurry pixels are not a problem.

**Comment by chuchana** [comment]
I meant something like "some more books" at the end of the page, because I thought it might clash with the visuals of the rest of the page.

**Comment by eudoxia0** [comment]
Hmm. Possibly a section with smaller covers for older, less currently useful or more domain-specific books.

**Comment by phoe** [comment]
The two recent books, Domkin's ProgAlgs and my TCLCS ,need to get added, too.
